docs/docs/data_classification_policy.rst

```diff
@@ -0,0 +1,93 @@
+.. _llm_as_judge:
+
+.. note::
+
+   To use this tutorial, you need to :ref:`install unitxt <install_unitxt>`.
+
+=====================================
+Sensitive data in unitxt ✨
+=====================================
+
+The section discusses how to properly handle sensitive data in unitxt in order to avoid accidentally sending propriety/confidential data to external services.
+
+Data classification policy
+----------------------------
+Whenever processing streams in unitxt, users can specify a parameter called `data_classification_policy`, which determines how this data should be treated. The taxonomy is defined by a user and may encompass multiple different policies.
+
+Each component used in unitxt (metrics, operators etc.) has the same parameter as well. This allows a given component to verify if it can process instances of a stream.
```

Comment:
```suggestion
Each component used in Unitxt (metrics, operators, inference engines etc.) has the same parameter as well. This allows a given component to verify if it can process instances of a stream.
```